misc/ci/macos-aarch64-build-release.ini

```diff
@@ -0,0 +1,51 @@
+[constants]
+# apple silicon only supports MacOS 11.0, so just use that
+macos_min = '11.1'
+cflags = ['-mmacosx-version-min='+macos_min, '-arch', 'arm64']
+ldflags = cflags
+
+[host_machine]
+cpu_family = 'aarch64'
+cpu = 'aarch64'
+system = 'darwin'
+endian = 'little'
+
+[binaries]
+c = 'clang'
+cpp = 'clang++'
+objc = 'clang'
+strip = 'strip'
+pkgconfig = 'pkg-config'
+
+[properties]
+needs_exe_wrapper = false
```

Comment:
Careful with that. Might need to set this to `true` to prevent meson from running test binaries when cross-compiling (which shouldn't work, unless the build machine is also aarch64). Note that this doesn't affect native builds.